app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -21,7 +21,9 @@ class MainActivity: BaseActivity() {
     private var atHome = true
 
     private val preferenceManager: PreferenceManager = PreferenceManager()
-
+    
+    private var doubleBackToExitPressedOnce = false
```

Comment:
cc @m-murad @isabelcosta Can we do it in a better way than using a global variable? I'm not sure about this.